cli/utils/index.ts

```diff
@@ -29,12 +29,29 @@ export const assertIsNonEmptyString = (str: string, varName: string) => {
   }
 };
 
+export const assertIsISOString = (str: string) => {
+  let parsedDate = new Date(Date.parse(str))
+
+  if(parsedDate.toISOString() !== str) {
+    throw new Error(`${str} is not a valid ISO timestamp. The ISO format is: YYYY-MM-DDTHH:mm:ss.sssZ`)
+  }
+}
```

Comment:
❓Right now this function is very strict 😬. The only format of ISO timestamps valid right now is `YYYY-MM-DDTHH:mm:ss.sssZ` which requires milliseconds to be entered.

Is there a standard timestamp format that forta uses? I was thinking of s valid time stamp be a modified ISO `YYYY-MM-DDTHH:mmZ` .

Reply:
I defer to @haseebrabbani on preference there, the API can be strict but we have choices on what's easiest for a user here on the cli.